tirgei/stack-overflow-lite
# #162601352 A user can login

#### What does this PR do?
This Pull Request creates a login page though which a user can sign in to the platform

#### Description of Task to be completed?
The tasks for this Pull Request:
1. Create a login form with username and password fields
2. Style the login page using CSS

#### How should this be manually tested?
Clone this repository on your machine:
```$ git clone https://github.com/tirgei/stack-overflow-lite.git```

CD into the project folder on your machine
```$ cd stack-overflow-lite```

Check out the branch for this PR
```$ git branch ft-user-login-162601352```

Open the `login.html` file on your browser

#### Pivotal Tracker story associated with this Pull Request?
[#162601352](https://www.pivotaltracker.com/story/show/162601352)

#### Screenshots
![login-ui](https://user-images.githubusercontent.com/17829576/49893816-64b81780-fe5d-11e8-9f6f-683275b88351.png)
